<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalcWindow</class>
 <widget class="QMainWindow" name="CalcWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>475</width>
    <height>389</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>475</width>
    <height>389</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>475</width>
    <height>389</height>
   </size>
  </property>
  <property name="font">
   <font/>
  </property>
  <property name="windowTitle">
   <string>CalcWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
  background-color: rgb(239, 239, 239);
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_graph">
    <property name="geometry">
     <rect>
      <x>135</x>
      <y>65</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(147, 146, 148);
  border: 1px solid gray;
  border-radius: 5px;
  color: white; 
}

QPushButton:pressed {
    background-color: rgb(198, 201, 203);
}</string>
    </property>
    <property name="text">
     <string>graph</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_mod">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>130</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(198, 201, 203);
  border: 1px solid gray;
  border-radius: 5px;
  color: black;
}

QPushButton:pressed {
    background-color: rgb(228, 226, 229);
}</string>
    </property>
    <property name="text">
     <string>mod</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_x">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>65</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(147, 146, 148);
  border: 1px solid gray;
  border-radius: 5px;
  color: white; 
}

QPushButton:pressed {
    background-color: rgb(198, 201, 203);
}</string>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ln">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>260</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(198, 201, 203);
  border: 1px solid gray;
  border-radius: 5px;
  color: black;
}

QPushButton:pressed {
    background-color: rgb(228, 226, 229);
}</string>
    </property>
    <property name="text">
     <string>ln</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_sin">
    <property name="geometry">
     <rect>
      <x>135</x>
      <y>195</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(198, 201, 203);
  border: 1px solid gray;
  border-radius: 5px;
  color: black;
}

QPushButton:pressed {
    background-color: rgb(228, 226, 229);
}</string>
    </property>
    <property name="text">
     <string>sin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_asin">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>195</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(198, 201, 203);
  border: 1px solid gray;
  border-radius: 5px;
  color: black;
}

QPushButton:pressed {
    background-color: rgb(228, 226, 229);
}</string>
    </property>
    <property name="text">
     <string>asin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_log">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>195</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(198, 201, 203);
  border: 1px solid gray;
  border-radius: 5px;
  color: black;
}

QPushButton:pressed {
    background-color: rgb(228, 226, 229);
}</string>
    </property>
    <property name="text">
     <string>log</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_cos">
    <property name="geometry">
     <rect>
      <x>135</x>
      <y>260</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(198, 201, 203);
  border: 1px solid gray;
  border-radius: 5px;
  color: black;
}

QPushButton:pressed {
    background-color: rgb(228, 226, 229);
}</string>
    </property>
    <property name="text">
     <string>cos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_acos">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>260</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(198, 201, 203);
  border: 1px solid gray;
  border-radius: 5px;
  color: black;
}

QPushButton:pressed {
    background-color: rgb(228, 226, 229);
}</string>
    </property>
    <property name="text">
     <string>acos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_tan">
    <property name="geometry">
     <rect>
      <x>135</x>
      <y>325</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(198, 201, 203);
  border: 1px solid gray;
  border-radius: 5px;
  color: black;
}

QPushButton:pressed {
    background-color: rgb(228, 226, 229);
}</string>
    </property>
    <property name="text">
     <string>tan</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_sqrt">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>325</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(198, 201, 203);
  border: 1px solid gray;
  border-radius: 5px;
  color: black;
}

QPushButton:pressed {
    background-color: rgb(228, 226, 229);
}</string>
    </property>
    <property name="text">
     <string>sqrt</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_atan">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>325</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(198, 201, 203);
  border: 1px solid gray;
  border-radius: 5px;
  color: black;
}

QPushButton:pressed {
    background-color: rgb(228, 226, 229);
}</string>
    </property>
    <property name="text">
     <string>atan</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_start">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>130</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(198, 201, 203);
  border: 1px solid gray;
  border-radius: 5px;
  color: black;
}

QPushButton:pressed {
    background-color: rgb(228, 226, 229);
}</string>
    </property>
    <property name="text">
     <string>(</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_end">
    <property name="geometry">
     <rect>
      <x>135</x>
      <y>130</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(198, 201, 203);
  border: 1px solid gray;
  border-radius: 5px;
  color: black;
}

QPushButton:pressed {
    background-color: rgb(228, 226, 229);
}</string>
    </property>
    <property name="text">
     <string>)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>130</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(198, 201, 203);
  border: 1px solid gray;
  border-radius: 5px;
  color: black;
}

QPushButton:pressed {
    background-color: rgb(228, 226, 229);
}</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>345</x>
      <y>260</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(198, 201, 203);
  border: 1px solid gray;
  border-radius: 5px;
  color: black;
}

QPushButton:pressed {
    background-color: rgb(228, 226, 229);
}</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_dot">
    <property name="geometry">
     <rect>
      <x>215</x>
      <y>325</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(198, 201, 203);
  border: 1px solid gray;
  border-radius: 5px;
  color: black;
}

QPushButton:pressed {
    background-color: rgb(228, 226, 229);
}</string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_minus">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>260</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(169, 217, 215);
  border: 1px solid gray;
  border-radius: 5px;
  color:black;
}

QPushButton:pressed {
    background-color: rgb(90, 186, 203);
}</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>260</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(198, 201, 203);
  border: 1px solid gray;
  border-radius: 5px;
  color: black;
}

QPushButton:pressed {
    background-color: rgb(228, 226, 229);
}</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>215</x>
      <y>130</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(198, 201, 203);
  border: 1px solid gray;
  border-radius: 5px;
  color: black;
}

QPushButton:pressed {
    background-color: rgb(228, 226, 229);
}</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>345</x>
      <y>130</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(198, 201, 203);
  border: 1px solid gray;
  border-radius: 5px;
  color: black;
}

QPushButton:pressed {
    background-color: rgb(228, 226, 229);
}</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ac">
    <property name="geometry">
     <rect>
      <x>215</x>
      <y>65</y>
      <width>125</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(147, 146, 148);
  border: 1px solid gray;
  border-radius: 5px;
  color: white; 
}

QPushButton:pressed {
    background-color: rgb(198, 201, 203);
}</string>
    </property>
    <property name="text">
     <string>AC</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_0">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>325</y>
      <width>125</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(198, 201, 203);
  border: 1px solid gray;
  border-radius: 5px;
  color: black;
}

QPushButton:pressed {
    background-color: rgb(228, 226, 229);
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_plus">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>195</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(169, 217, 215);
  border: 1px solid gray;
  border-radius: 5px;
  color:black;
}

QPushButton:pressed {
    background-color: rgb(90, 186, 203);
}</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>195</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(198, 201, 203);
  border: 1px solid gray;
  border-radius: 5px;
  color: black;
}

QPushButton:pressed {
    background-color: rgb(228, 226, 229);
}</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_1">
    <property name="geometry">
     <rect>
      <x>215</x>
      <y>260</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(198, 201, 203);
  border: 1px solid gray;
  border-radius: 5px;
  color: black;
}

QPushButton:pressed {
    background-color: rgb(228, 226, 229);
}</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>215</x>
      <y>195</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(198, 201, 203);
  border: 1px solid gray;
  border-radius: 5px;
  color: black;
}

QPushButton:pressed {
    background-color: rgb(228, 226, 229);
}</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_div">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>65</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(169, 217, 215);
  border: 1px solid gray;
  border-radius: 5px;
  color:black;
}

QPushButton:pressed {
    background-color: rgb(90, 186, 203);
}</string>
    </property>
    <property name="text">
     <string>÷</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_mul">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>130</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(169, 217, 215);
  border: 1px solid gray;
  border-radius: 5px;
  color:black;
}

QPushButton:pressed {
    background-color: rgb(90, 186, 203);
}</string>
    </property>
    <property name="text">
     <string>×</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>345</x>
      <y>195</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(198, 201, 203);
  border: 1px solid gray;
  border-radius: 5px;
  color: black;
}

QPushButton:pressed {
    background-color: rgb(228, 226, 229);
}</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_equl">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>325</y>
      <width>61</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(169, 217, 215);
  border: 1px solid gray;
  border-radius: 5px;
  color:black;
}

QPushButton:pressed {
    background-color: rgb(90, 186, 203);
}</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_back">
    <property name="geometry">
     <rect>
      <x>345</x>
      <y>65</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(147, 146, 148);
  border: 1px solid gray;
  border-radius: 5px;
  color: white; 
}

QPushButton:pressed {
    background-color: rgb(198, 201, 203);
}</string>
    </property>
    <property name="text">
     <string>&lt;-</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="result_show">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>8</y>
      <width>465</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>23</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(251, 251, 251);
color:black;
border: 1px solid gray;
border-radius: 5px;</string>
    </property>
    <property name="maxLength">
     <number>255</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_pow">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>65</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(198, 201, 203);
  border: 1px solid gray;
  border-radius: 5px;
  color: black;
}

QPushButton:pressed {
    background-color: rgb(228, 226, 229);
}</string>
    </property>
    <property name="text">
     <string> xʸ </string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
